sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -106,23 +106,21 @@ def subscribe_port_config_change():
         sel.addSelectable(port_tbl)
     return sel, asic_context
 
-def subscribe_port_update_event(db_list=['APPL_DB', 'STATE_DB']):
-    port_tbl_map = {
-        'APPL_DB': swsscommon.APP_PORT_TABLE_NAME,
-        'CONFIG_DB': swsscommon.CFG_PORT_TABLE_NAME,
-        'STATE_DB': 'TRANSCEIVER_INFO'
-    }
+def subscribe_port_update_event():
+    port_tbl_map = [
+        {'APPL_DB': swsscommon.APP_PORT_TABLE_NAME},
+        {'STATE_DB': 'TRANSCEIVER_INFO'},
+        {'STATE_DB': 'PORT_TABLE'},
+    ]
 
     sel = swsscommon.Select()
     asic_context = {}
     namespaces = multi_asic.get_front_end_namespaces()
-    for db_name in db_list:
-        if db_name not in port_tbl_map:
-            continue
+    for d in port_tbl_map:
```

Comment:
As indicated earlier, do we really want to have all these listeners registered all the time?

Reply:
@ds952811 yes, for these events:-

1. host_tx_ready coming from PORT_TABLE of STATE_DB
2. TRANSCEIVER_INFO for detecting Xcvr presence change
3. APPL_DB for listening to configuration changes for breakout, shutdown/startup etc